MyProject/src/main/java/com/netcracker/config/ConfigFromYaml.java

```diff
@@ -0,0 +1,17 @@
+package com.netcracker.config;
+
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+@Component
+@ConfigurationProperties("arangodb")
+@Data
+public class ConfigFromYaml {
```

Comment:
It feels that having this separate config is excessive. Please remove this class, and inject all db-related parameters from yaml inside DBConfig configuration bean by using @Value annotation.